### PR TITLE
Add GC references for my PhD upgrade report.

### PR DESCRIPTION
**Pull request by jacob-hughes**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2008,3 +2008,55 @@ @article{turner77error
     month = Aug,
     doi = "10.1016/0020-0190(77)90023-0"
 }
+
+@article{mccarthy60recursive,
+    author = "McCarthy, John",
+    title = "Recursive Functions of Symbolic Expressions and Their Computation by Machine, Part I",
+    journal = "Commun. ACM",
```
Comment (ltratt) [inline softdev.bib]: I think we call this "CACM" elsewhere?

File: softdev.bib [hunk 2]
```diff
@@ -2008,3 +2008,55 @@ @article{turner77error
...
+    number={2},
+    pages={291--307},
+    year={1970},
+    month=Feb
+}
+
+@inproceedings{bacon2001concurrent,
+    title={Concurrent cycle collection in reference counted systems},
+    author={Bacon, David F and Rajan, Vadakkedathu T},
+    booktitle={European Conference on Object-Oriented Programming},
```
Comment (ltratt) [inline softdev.bib]: Just `ECOOP`.

File: softdev.bib [hunk 3]
```diff
@@ -2008,3 +2008,55 @@ @article{turner77error
...
+  title={The garbage collection handbook: the art of automatic memory management},
+  year={2016},
+  publisher={Chapman and Hall/CRC},
+  month=Sep
+}
+
+@article{boehm1988garbage,
+  author={Boehm, Hans-Juergen and Weiser, Mark},
+  title={Garbage collection in an uncooperative environment},
+  journal={Software: Practice and Experience},
```
Comment (ltratt) [inline softdev.bib]: `SPE`

**Comment by ltratt** [comment]
Let's get all the indents to be 4 spaces.

File: softdev.bib (hunk 1 above)
Comment (jacob-hughes) [reply softdev.bib]: It's Commun.~ACM elsewhere, I've added the tilde.

**Comment by jacob-hughes** [comment]
fixed

**Comment by ltratt** [comment]
Looks good. Please squash.

**Comment by jacob-hughes** [comment]
Squashed!

**Pull request merged**